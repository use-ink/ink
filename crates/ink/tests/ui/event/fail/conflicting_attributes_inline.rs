#[ink::event]
#[ink(anonymous, signature_topic = "1111111111111111111111111111111111111111111111111111111111111111")]
pub struct Event {
    pub topic: [u8; 32],
}

fn main() {}
