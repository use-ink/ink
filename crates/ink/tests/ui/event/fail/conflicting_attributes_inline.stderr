error: cannot specify `signature_topic` with `anonymous` in ink! event item configuration argument
 --> tests/ui/event/fail/conflicting_attributes_inline.rs:2:18
  |
2 | #[ink(anonymous, signature_topic = "1111111111111111111111111111111111111111111111111111111111111111")]
  |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
